### PR TITLE
Contact - Minor Layout Adjustments

### PR DESCRIPTION
**Pull request by cameron-freshworks**
*Issue #:* /bcgov/entity#10687

*Description of changes:*
* Updated some layouts to meet spacing requirements

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/ContactInfo/ContactInfo.vue [hunk 1]
```diff
@@ -186,7 +186,7 @@
             >
             </v-text-field>
           </v-col>
-          <v-col>
+          <v-col class="ml-2">
```
Comment (severinbeauvais) [inline src/components/ContactInfo/ContactInfo.vue:189]: Won't this cause it to not line up with other columns on the page?

Comment (cameron-freshworks) [reply src/components/ContactInfo/ContactInfo.vue:189]: This is the split between the two input fields. The cols are on auto, so splitting the margin between the two elements keeps it equal. 

Screen Shot 2022-03-04 at 10 01 02 AM

**Review by severinbeauvais — APPROVED** [review]

File: src/components/ContactInfo/ContactInfo.vue (hunk 1 above)
Comment (severinbeauvais) [reply src/components/ContactInfo/ContactInfo.vue:189]: Oh those fields. I thought this was changing the main columns 1 and 2.

Comment (cameron-freshworks) [reply src/components/ContactInfo/ContactInfo.vue]: Updated the approach to use just padding on the Col 1 and let the auto cols handle the second. 
Works the same, looks the same. Styling in 1 place though 👍

Comment (severinbeauvais) [reply src/components/ContactInfo/ContactInfo.vue]: When the phone/extension columns wrap, is the second column indented due to the left margin?

Comment (severinbeauvais) [reply src/components/ContactInfo/ContactInfo.vue]: Oh nvm -- I see you latest code has only right margin on the first column, so the second column should be fine.

**Pull request merged**